sdk/tests/js/webgl-test-utils.js

```diff
@@ -739,7 +739,16 @@ var glTypeToTypedArrayType = function(gl, type) {
     case gl.INT:
       return window.Int32Array;
     case gl.UNSIGNED_INT:
+    case gl.UNSIGNED_INT_5_9_9_9_REV
+    case gl.UNSIGNED_INT_10F_11F_11F_REV:
+    case gl.UNSIGNED_INT_2_10_10_10_REV:
+    case gl.UNSIGNED_INT_24_8:
       return window.Uint32Array;
+    case gl.HALF_FLOAT:
+    case gl.FLOAT:
+      return window.Float32Array;
+    case gl.FLOAT_32_UNSIGNED_INT_24_8_REV:
```

Comment:
Again, no combined typed array for both float and uint, so I use Float64Array...